### ISSUE TITLE
Community documentation

### ISSUE THREAD

**Issue by dedok** [tarantool/doc#34]
Any people should have a chance to contribute our documentation.

Let's have a look at the nginx community (https://www.nginx.com/resources/wiki/)
Anyone may change anything in the nginx documentation.

For instance, If I want to change something in the docs, I'll need push one button -- easy and useful! This action will bring me at the GH where I could make some changes into the docs and do a PR to main repo.

**Comment by lenkis** [comment]
I've discussed this gh item with the team. Here's what we suggest:

Although we're quite happy with the current markdown-based workflow -- in fact, our team intentionally migrated to markdown long before I joined the project, because they considered markdown friendly enough to edit -- we like the idea to streamline user's feedback. As a possible solution, we can enable online commenting for each doc page. @dedok, does this sound reasonable for you?

We agreed to assign @Sulverus as implementor (adding the commenting feature requires web development).
